tests/tests_app/core/test_lightning_api.py

```diff
@@ -195,9 +196,9 @@ def test_update_publish_state_and_maybe_refresh_ui():
     publish_state_queue = _MockQueue("publish_state_queue")
     api_response_queue = _MockQueue("api_response_queue")
 
-    publish_state_queue.put(app.state_with_changes)
+    publish_state_queue.put((app.state_with_changes, None))
 
-    thread = UIRefresher(publish_state_queue, api_response_queue)
+    thread = UIRefresher(publish_state_queue, api_response_queue, None)
```

Comment:
perhaps we could use named arguments here for clarity.

Reply:
Whoops, that was there by accident, removed :smiley: